helm/README.md

```diff
@@ -0,0 +1,15 @@
+# Nessie Helm chart
+
+## Installation
```

Comment:
can this be filled out to describe the `Values.yaml` settings and what options a user has at install time?

Reply:
yep will add more docs around configurable things

Reply:
done